build.sbt

```diff
@@ -57,6 +57,7 @@ lazy val commonSettings = Seq(
     "-unchecked",
   ),
```

Comment:
Merge commit: it must include a note that this a Java rewrite of the code that was previously in Scala. I'd also add a permalink to the repo before this was merged, so that you can navigate back and see the code history somehow.